instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -194,7 +194,11 @@ func (c *config) handleRPC(ctx context.Context, rs stats.RPCStats) {
 		span.End()
 
 		metricAttrs = append(metricAttrs, rpcStatusAttr)
-		c.rpcDuration.Record(wctx, float64(rs.EndTime.Sub(rs.BeginTime)), metric.WithAttributes(metricAttrs...))
+
+		// Use floating point division here for higher precision (instead of Millisecond method).
+		elapsedTime := float64(rs.EndTime.Sub(rs.BeginTime)) / float64(time.Millisecond)
```

Comment:
How about doing the same floating point division in `interceptor.go`?

Reply:
Addressed in 4b3e369d676a7bcd113b393df36d62a9215fe177

Reply:
Please suggest a change to the CHANGELOG if you feel it is necessary.